python/tests/ecl_tests/test_sum.py

```diff
@@ -23,7 +23,12 @@
 import cwrap
 import stat
 import pandas
-from pandas.testing import assert_frame_equal
+
+try:
+    from pandas.testing import assert_frame_equal
+except ImportError:
+    assert_frame_equal = None
```

Comment:
I'm a bit uncomfortable with skipping the test for older pandas versions. The `DataFrame.equals` method is much older. How about defining `assert_frame_equal` as :
```python
def assert_frame_equal(a, b):
    if not a.equals(b):
        raise AssertionError('Expected dataframes to be equal')

In case it is missing?
```

Reply:
Good idea - done!